wirecard/qmore.py

```diff
@@ -74,9 +74,9 @@ def init_datastorage(self, orderIdent,
         return result
 
     def init_frontend(self, amount, currency, paymentType, language,
-                      orderDescription, successUrl, cancelUrl, failureUrl,
-                      serviceUrl, confirmUrl, consumerUserAgent,
-                      consumerIpAddress, autoDeposit=None,
+                      orderDescription, customerStatement, successUrl,
```

Comment:
The way I understand it is that customerStatement is not a required value for Wirecard. It's optional. So we should also make it optional in the init_frontend function. We can do that by making it a keyword argument with a default value of None. That way if you use the library you don't have to pass the CustomerStatement argument if you don't need it.